utils/admin_console_index.jsx

```diff
@@ -75,16 +77,23 @@ export function adminDefinitionsToUrlsAndTexts(adminDefinition, intl) {
     return entries;
 }
 
-export function generateIndex(adminDefinition, intl) {
+export function generateIndex(AdminDefinition, intl, plugins) {
```

Comment:
err feels a bit awkward to not have two similar definitions not close to each other, can we change this to `generateIndex(adminDefinition, plugins, intl)` that way the definitions are close to each other.